src/card/card-header.jsx

```diff
@@ -94,14 +94,15 @@ const CardHeader = React.createClass({
     let titleStyle = this.prepareStyles(styles.title, this.props.titleStyle);
     let subtitleStyle = this.prepareStyles(styles.subtitle, this.props.subtitleStyle);
 
-    let avatar = this.props.avatar;
+    let avatar = this.props.avatar || null;
```

Comment:
I don't think this change solve the problem. As I tried, the avatar 'dark circle' is still there.